mlflow/gateway/providers/togetherai.py

```diff
@@ -308,12 +309,26 @@ def base_url(self):
         return "https://api.together.xyz/v1"
 
     @property
-    def auth_headers(self):
+    def headers(self):
         return {"Authorization": f"Bearer {self.togetherai_config.togetherai_api_key}"}
 
+    @property
+    def adapter(self):
+        return TogetherAIAdapter
```

Comment:
Is this an abstract property?

Reply:
Nope, but can do in a follow-up. See https://github.com/mlflow/mlflow/pull/13717#discussion_r1835934794 for why I didn't add it in this PR.